easybuild/easyconfigs/o/ORCA/ORCA-4.0.1-OpenMPI-2.0.2.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "PackedBinary"
+
+name = "ORCA"
+version = '4.0.1'
+
+ompi_ver = '2.0.2'
+versionsuffix = '-OpenMPI-%s' % ompi_ver
+
+homepage = 'http://cec.mpg.de/forum/'
+description = """ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry
+ with specific emphasis on spectroscopic properties of open-shell molecules.
+  It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single-
+ and multireference correlated ab initio methods.
+ It can also treat environmental and relativistic effects."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# Download from https://cec.mpg.de/orcadownload/index.php
+sources = ['%%(namelower)s_%s_linux_x86-64_openmpi%s.tar.xz' % (version.replace('.', '_'), ompi_ver.replace('.', ''))]
+
+checksums = ['49f4e155a5425d9be2b1e90bf2fa7b61']
```

Comment:
@hajgato We're slowly moving towards SHA256 checksum, so let's replace this with:

```python
checksums = ['6644b95fdb16e117b870fd9e5cea8b4f829f1547d5df4e87d52ef08e3f1ee310']
```

(do verify on your end)